src/content/docs/fr/guides/integrations-guide/alpinejs.mdx

```diff
@@ -0,0 +1,129 @@
+---
+type: integration
+title: '@astrojs/alpinejs'
+description: Apprenez à utiliser l'intégration du framework @astrojs/alpinejs pour étendre le support des composants dans votre projet Astro.
+githubIntegrationURL: 'https://github.com/withastro/astro/tree/main/packages/integrations/alpinejs/'
+category: renderer
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+Cette **[intégration Astro][astro-integration]** ajoute [Alpine.js](https://alpinejs.dev/) à votre projet afin que vous puissiez utiliser Alpine.js n'importe où sur votre page.
+## Installation
```

Comment:
Need space just before this line (this don't affect the final render of the page)
```suggestion

## Installation
```